### PR TITLE
feat: preview vetoken balance after lock or increase

### PR DESCRIPTION
**Pull request by wjw12**
for UI integration purpose

### CODE REVIEW

File: sources/vetoken.move [hunk 1]
```diff
@@ -409,6 +409,35 @@ module vetoken::vetoken {
...
+    #[view]
+    public fun locked_coin_amount<CoinType>(account_addr: address): u64 acquires VeTokenStore {
+        if (!is_account_registered<CoinType>(account_addr)) return 0;
+        let vetoken_store = borrow_global<VeTokenStore<CoinType>>(account_addr);
+        coin::value(&vetoken_store.vetoken.locked)
+    }
+
+    #[view]
+    public fun preview_balance_after_increase<CoinType>(account_addr: address, added_amount: u64, increment_epochs: u64): u64 acquires VeTokenInfo, VeTokenStore {
+        //assert!(!can_unlock<CoinType>(account_addr), ERR_VETOKEN_);
```
Comment (wjw12) [inline sources/vetoken.move:421]: TODO: need to merge

File: sources/vetoken.move [hunk 2]
```diff
@@ -425,6 +425,29 @@ module vetoken::vetoken {
...
+        assert!(!can_unlock<CoinType>(account_addr), ERR_VETOKEN_NOT_LOCKED);
+        let now_epoch = now_epoch<CoinType>();
+        let locked_amount = locked_coin_amount<CoinType>(account_addr);
+        // If there's no existing lock, it's equivalent to having a locked position of zero amount and unlockalbe epoch of now_epoch
+        let old_unlockable_epoch = if (locked_amount == 0) {
+            now_epoch
+        }
+        else {
+            unlockable_epoch<CoinType>(account_addr)
+        };
```
Comment (0xbe1) [inline sources/vetoken.move:440]: prefer one-line it

File: sources/vetoken.move [hunk 3]
```diff
@@ -425,6 +425,29 @@ module vetoken::vetoken {
         vetoken_info.max_locked_epochs
     }
 
+    #[view]
+    public fun preview_balance_after_increase<CoinType>(account_addr: address, added_amount: u64, increment_epochs: u64): u64 acquires VeTokenInfo, VeTokenStore {
+        // If there's an existing unlockable position, user must unlock before increasing
+        assert!(!can_unlock<CoinType>(account_addr), ERR_VETOKEN_NOT_LOCKED);
+        let now_epoch = now_epoch<CoinType>();
+        let locked_amount = locked_coin_amount<CoinType>(account_addr);
+        // If there's no existing lock, it's equivalent to having a locked position of zero amount and unlockalbe epoch of now_epoch
```
Comment (0xbe1) [inline sources/vetoken.move:434]: unlockalbe -> unlockable

**Review by 0xbe1 — APPROVED** [review]
lgtm

**Pull request merged**